ast2vec/repo2/base.py

```diff
@@ -2,18 +2,19 @@
 import logging
 import multiprocessing
 import os
-from queue import Queue
 import re
 import shutil
 import subprocess
 import tempfile
 import threading
 from collections import namedtuple
+from queue import Queue
```

Comment:
It was discussed and unfortunately missed because I did not check Alfredo: https://github.com/src-d/guide/pull/52#discussion_r129808242